frontend/src/pages/Registration/Registration.styles.tsx

```diff
@@ -1 +1,79 @@
 import styled from 'styled-components';
+
+const Background = styled.div`
+
+    --group-spacing: 25px;
+    --checkbox-spacing: 5px;
+
+    background: rgb(254,254,254);
+    background: linear-gradient(0deg, rgba(254,254,254,1) 0%, rgba(203,218,239,1) 51%, rgba(254,254,254,1) 100%);
+    text-align: left;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: #1E2354;
```

Comment:
Incase we are re-using the colours we can move it to theme.ts

Reply:
kk I will do that